configure

```diff
@@ -248,7 +248,10 @@ function detect_machine {
         soc="AM33XX"
         ;;
     *)
-        soc="unknown"
```

Comment:
Shouldn't we keep this (but in an else statement after trying to detect rockpi), in case the soc cannot be detected?

Reply:
It can be kept there. However _soc_ variable is already initialized:
https://github.com/mysensors/MySensors/blob/eed25da65c6b8ba4c52262312e3b5304182ef859/configure#L154-L155